forc-fs/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "forc-fs"
+version = "0.31.3"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+homepage = "https://fuel.network/"
+license = "Apache-2.0"
+repository = "https://github.com/FuelLabs/sway"
+description = "Filesystem abstraction for Forc"
+
+[dependencies]
+file-lock = "2.1"
```

Comment:
Out of curiosity, what prompted the choice to use `file-lock` here? It looks like there are a few alternatives that have a lot more use like [fslock](https://crates.io/crates/fslock) or [fd-lock](https://crates.io/crates/fd-lock) - maybe it's worth switching to one of these more established options?

Reply:
Ahh I saw fslock but not fd-lock. `file-lock` seemed at the time to me was more mature than `fslock`, but it seems like `fd-lock` is a better alternative since its from a core maintainer. Will re-write using that.